.github/workflows/v2beta.deployNodeAppToAzureWebApp.yml

```diff
@@ -0,0 +1,24 @@
+on:
```

Comment:
As mentioned in the previous PR, can we use a single workflow moving forward to prevent multiple runs from being created on pushes? I think it would be best if one run with multiple jobs contained within it was spawned from a push.

Reply:
I have been thinking again after our earlier discussion. We want these workflows to be a showcase for us as well as to point end users to them. Having a single workflow would make it look big and might confuse end users. for example, a Node developer looking at the workflow doesn't need to know about building Python or DotNetCore apps.

Reply:
That's a good point -- looking at the actions page for the repository, users can filter out runs by the workflow, so that makes it easier to scope down runs that a user on a specific platform would want to look at. The only concern I would have is the ballooning number of runs that appear on our actions page if we keep increasing the number of workflows we have, but the ability for users to filter out filters they don't want to look at makes it essentially a non-factor for me.

Reply:
Yeah, let us see how this pattern goes. Ideally we want to show E2E scenario in simple steps so that users' first time experience is good. Do you have any other feedback?